docs/pages/eas-update/build-locally.mdx

```diff
@@ -195,11 +203,11 @@ Now we're ready to publish an update that will include our changes.
 
 After optionally making additional changes to **App.js**, publish an update with the following command:
 
-<Terminal cmd={['$ eas update']} />
+<Terminal cmd={['$ eas update']} cmdCopy="eas update" />
```

Comment:
```suggestion
<Terminal cmd={['$ eas update']} />
```

Since this is a single command, no need to add manually add `cmdCopy`. All single line values for this component have copy functionality enabled by default.